examples/common.ts

```diff
@@ -481,7 +486,75 @@ const deployWeiroll = async (provider: ethers.JsonRpcProvider) => {
+const makeTransaction = async (
+  signer: ethers.Signer,
+  txData: ethers.TransactionRequest,
+  ctx?: string
+) => {
+  const tx = await signer.sendTransaction(txData);
+  const receipt = await tx.wait();
+  if (receipt?.status === 0)
+    throw new Error(`transaction failed: ${ctx || ''}`);
+  console.log(`tx succeeded: ${ctx}: ${receipt!.hash}`);
+  return receipt!;
+};
+
+// allow `toHope` to control `user`'s position
+export const vatHope = async (
+  provider: ethers.JsonRpcProvider,
+  user: string,
+  toHope: string
```

Comment:
```suggestion
  who: string,
  usr: string
```
nit: try to keep vars aligned with maker's smart contracts